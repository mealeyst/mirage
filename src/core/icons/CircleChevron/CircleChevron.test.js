import React from 'react'
import 'jest-styled-components'

import { CircleChevron } from 'SRC'

const { mountWithTheme } = global

describe('(Styled Component) CircleChevron', () => {
  const createCircleChevron = (props) => {
    return mountWithTheme(<CircleChevron left {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createCircleChevron())
    .toMatchSnapshot()
  })
})
