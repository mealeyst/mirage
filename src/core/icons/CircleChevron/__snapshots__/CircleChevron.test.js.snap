// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled Component) CircleChevron matching the snapshot 1`] = `
.c1 polyline {
  stroke-linecap: round;
  fill: none;
}

.c0 ellipse {
  fill: #76C2F3;
}

.c0 polyline {
  stroke: #FFFFFF;
  stroke-width: 4;
}

<Styled(Styled(CircleChev))
  left={true}
>
  <Styled(CircleChev)
    className="c0"
    left={true}
  >
    <CircleChev
      className="c0 c1"
      left={true}
      width="4rem"
    >
      <svg
        className="c0 c1"
        viewBox="0 0 60 60"
      >
        <ellipse
          cx="30"
          cy="30"
          rx="30"
          ry="30"
        />
        <polyline
          className="left"
          points="36.5,43.2 23.5,30 36.5,16.8"
        />
      </svg>
    </CircleChev>
  </Styled(CircleChev)>
</Styled(Styled(CircleChev))>
`;
